plugins/file-open-save-new/core/src/main/java/org/pentaho/di/plugins/fileopensave/providers/local/LocalFileProvider.java

```diff
@@ -168,15 +168,33 @@ public List<LocalFile> delete( List<LocalFile> files, VariableSpace space ) {
     List<LocalFile> deletedFiles = new ArrayList<>();
```

Comment:
@krishnaalluhitachi Please reformat your changes in this file to add the expected whitespace. You can find instructions here https://github.com/pentaho/pentaho-coding-standards